src/main/kotlin/io/provenance/client/wallet/WalletSigner.kt

```diff
@@ -10,16 +10,17 @@ import io.provenance.hdwallet.wallet.Account
 import io.provenance.hdwallet.wallet.Wallet
 
 enum class NetworkType(
-    /**
-     * The hrp (Human Readable Prefix) of the network address
-     */
-    val prefix: String,
-    /**
-     * The HD wallet path
-     */
-    val path: String
+        /**
+         * The hrp (Human Readable Prefix) of the network address
+         */
+        val prefix: String,
+        /**
+         * The HD wallet path
+         */
+        val path: String
 ) {
     TESTNET("tp", "m/44'/1'/0'/0/0'"),
```

Comment:
its unfortunate we had testnet hardened and mainnet unhardened since its super confusing looking at name thats what is happening - then we could have suffixed it accordingly

Reply:
Agreed.